echopype/convert/set_groups_azfp6.py

```diff
@@ -0,0 +1,893 @@
+"""
+Class to save unpacked echosounder data to appropriate groups in netcdf or zarr.
+"""
+
+from typing import List
+
+import numpy as np
+import xarray as xr
+
+from ..utils.coding import set_time_encodings
+from .set_groups_base import SetGroupsBase
+
+
+class SetGroupsAZFP6(SetGroupsBase):
```

Comment:
@dash-uvic : It seems that a lot of the code here are copy-pasted here from `SetGroupsAZFP`? If that's the case, why don't you inherit from that and can only change the parts that are different? (Note I have not gone in detail to compare what was changed).

Reply:
Since the hardware is still in flux (so Sv values, battery calculations, etc. may change) I decided to keep it separate for now, and then create a base class both ULS5/6 (both in groups and converter classes) would inherit from if we end up going with the current setup.  You're right, the AZFP6 group has not changed a lot, and I don't think it'll change much in the future so I rewrite that to inherit from the original AZFP group

Reply:
To make sure I understand, the approach you are taking is to have 2 completely separate class for now and then will merge the common components to have a base class for both after the hardware is settled?

Reply:
Yes, that's my intention.  Or if not a base class, inherit from the current AZFP class but as the class structure will remain the same, even if the internal function changes, so it makes sense to abstract and inherit.  If there is a preference for how you'd like the code structure, I'm flexible.

Reply:
I think both would be fine. It seems that inheriting from a common based class into separate ULS5/6 classes would make it pretty extensible (not that anyone wants to think about more changes at the moment... lol), so we can go with that. :)